### PR TITLE
tree-shakable

### PR DESCRIPTION
**Pull request by ryoppippi**
Make tree-shakable .

After #34 is merged, rebase it and open this PR

### CODE REVIEW

File: browser-symbols.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const info = 'ℹ️';
+const success = '✅';
+const warning = '⚠️';
+const error = '❌️';
```
Comment (sindresorhus) [inline browser-symbols.js:4]: Easier to just export these directly.

File: symbols.d.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export const info: string;
+export const success: string;
+export const warning: string;
+export const error: string;
```
Comment (sindresorhus) [inline symbols.d.ts:4]: I don't see the need to have these in a separate file. Can just be in index.d.ts.

Comment (ryoppippi) [reply symbols.d.ts]: revert it!

**Comment by ryoppippi** [comment]
ping!

**Pull request merged**